cypress/integration/Common.feature

```diff
@@ -11,6 +11,9 @@ Feature: Test cases for common elements and functionalities
       | network          |
       | matic            |
       | xdai             |
+      #TODO Uncomment this once the test account is set on the new networks
```

Comment:
TODO

Reply:
Yeah, need to create streams etc on the new networks, will do that when adding the network tests on dashboard